python/tvm_ffi/access_path.py

```diff
@@ -39,11 +39,30 @@ class AccessKind(IntEnum):
 class AccessStep(core.Object):
     """Access step container."""
 
+    if TYPE_CHECKING:
+        kind: AccessKind
+        key: Any
+
 
 @register_object("ffi.reflection.AccessPath")
 class AccessPath(core.Object):
     """Access path container."""
 
+    if TYPE_CHECKING:
```

Comment:
ditto. We may want either a separate stub file via stub generation, or suppress type errors

Reply:
Does this need to depend on TYPE_CHECKING?